### PR TITLE
	Slovak language added > modified:   resources/messages/en/app.php

### PR DESCRIPTION
**Pull request by evan70**
modified:   resources/messages/id/app.php Slovak language added
	modified:   resources/messages/ru/app.php Slovak Language added
	new file:   resources/messages/sk/app.php Slovak language file
	new file:   resources/rbac/assignments.php Slovak language added
	modified:   views/layout/main.php Slovak language added

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

**Comment by evan70** [comment]
Slovak language added

**Comment by evan70** [comment]
Scrutinizer fails to build in step titled :

In LocalServer.php line 163:
                                                                               
Code coverage file /home/scrutinizer/build/runtime/tests/_output/c3tmp/code  
  coverage.serialized does not exist  

... ending up with message :
The coverage file '/home/scrutinizer/build/runtime/tests/_output/coverage.xml' was not found.

File: resources/rbac/assignments.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (xepozz) [inline resources/rbac/assignments.php:1]: Is it necessary?

**Review by xepozz — CHANGES_REQUESTED** [review]

**Comment by xepozz** [comment]
> Scrutinizer fails to build in step titled :
> 
> In LocalServer.php line 163:
> 
> Code coverage file /home/scrutinizer/build/runtime/tests/_output/c3tmp/code coverage.serialized does not exist
> 
> ... ending up with message : The coverage file '/home/scrutinizer/build/runtime/tests/_output/coverage.xml' was not found.

Don't worry about it.

**Comment by xepozz** [comment]
@evan70 why did you close it?

**Comment by evan70** [comment]
i was hungry maybe, it does't matter, one mistake follow another one :)
thX ;) there's much more time to push it later ...  sorry 4 bumpin'

**Pull request closed without merging**